### DISCUSSION TITLE
Flash Bootloader

### DISCUSSION THREAD

**Discussion by ra1n08** [NicoHood/HoodLoader2#125] [Q&A]
hi, i'm just have 1 arduino uno and i want to burn Hoodloader2 to 16u2, so i not have other uno or mega but i have a usb serial CH340

So, Does ch340 can burn Hoodloader2 to 16u2? using what program?

tks!

**Comment by NicoHood** [comment]
what? if you have an arduino with a 16u2, you dont need another one. But if you want to use an arduino with ONLY a CH340 that is not possible
